content/project-framing.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Framing the Project
+home_spotlights:
+  enabled: false
+  weight: 1
+  excerpt: The success of your project largely depends on understanding your nonprofit's project.
+      Before you crack open your IDE and dive into the code, take some time to glean important insights
+      up front, so that you can be more efficient with your time during the hack.
+  home_img:
+    path: images/framing-barn.jpg
+    data_position: center center
+    url: "#"
+layout: page
+---
+
+Congratulations! You've selected your nonprofit project and are ready to get started. Before you crack open your IDE and dive into the code, the first few hours of the jam are most well spent building a deep understanding of the problem your nonprofit is trying to solve. By doing this well, you will have a better defined path for the next 40+ hours. You may also discover unexpected solutions that your nonprofit didn't even know to ask for. After all, they are experts in serving their communities, not the nuances of the latest and greatest technology.
+
+## Interviewing Your Nonprofit
+
+
+
+### Take (Post It) Notes
+A lot is about to be said in the next hour. You cannot possibly remember it all. You also don't know enough yet to understand how what they are saying this moment connects to an insight you'll have in two hours. You could do this in a notebook or shared document, like your probably used to for school, or you could try something a bit different.
```

Comment:
```suggestion
A lot is about to be said in the next hour. You cannot possibly remember it all. You also don't know enough yet to understand how what they are saying this moment connects to an insight you'll have in two hours. Your first instinct may be to do this in a notebook or shared document, but the following, "low tech" approach has a number of benefits.
```

Reply:
if not the whole rewording suggestion, there is a typo "like your probably" -> "like you're probably"